lib/dialects/mysql/query.js

```diff
@@ -92,7 +92,22 @@ class Query extends AbstractQuery {
       this.handleInsertQuery(data);
 
       if (!this.instance) {
-        result = data[this.getInsertIdField()];
+        // handle bulkCreate AI primiary key 
+        if (
+          data.constructor.name === 'ResultSetHeader'
+          && this.model
+          && this.model.autoIncrementField
+          && this.model.autoIncrementField === this.model.primaryKeyAttribute
+          && this.model.rawAttributes[this.model.primaryKeyAttribute]
+        ) {
+          const startId = data[this.getInsertIdField()];
+          result = [];
+          for (let i = startId; i < startId + data.affectedRows; i++) {
```

Comment:
Found more reference:
http://dev.mysql.com/doc/refman/5.7/en/innodb-auto-increment-handling.html
> With innodb_autoinc_lock_mode set to 0 (“traditional”) or 1 (“consecutive”), the auto-increment values generated by any given statement will be consecutive, without gaps, because the table-level AUTO-INC lock is held until the end of the statement, and only one such statement can execute at a time.